projects/components/src/navigation/navigation-list-component.service.ts

```diff
@@ -43,9 +47,9 @@ export class NavigationListComponentService {
     return navItems.map(navItem => {
       if (navItem.type === NavItemType.Link) {
         return this.pageTimeRangePreferenceService.getTimeRangePreferenceForPage(navItem.matchPaths[0]).pipe(
-          map(timeRange => ({
+          map(resolveTimeRange => ({
             ...navItem,
-            timeRange: timeRange,
+            resolveTimeRange: resolveTimeRange,
```

Comment:
If keeping this design, can we rename this to timeRangeResolver (noun over verb for properties/variable)